docs/MO_DG/prepare_model/customize_model_optimizer/Extending_Model_Optimizer_with_Caffe_Python_Layers.md

```diff
@@ -20,23 +20,24 @@ layer {
-operation enabling pipeline and documentation on how to support other Caffe operations (written in C++) is described in
-the main document [Customize_Model_Optimizer](Customize_Model_Optimizer.md).
+This article describes only a procedure on how to extract operator attributes in the Model Optimizer. The rest of the
+operation enabling pipeline and information on how to support other Caffe operations (written in C++) is described in
+the [Customize_Model_Optimizer](Customize_Model_Optimizer.md) guide.
 
 ## Writing Extractor for Caffe Python Layer
-Custom Caffe Python layers have an attribute `type` (defining the type of the operation) equal to `Python` and two
-mandatory attributes `module` and `layer` in the `python_param` dictionary. The `module` defines the Python module name
-with the layer implementation, while `layer` value is an operation type defined by an user. In order to extract
+Custom Caffe Python layers have an attribute *`type`* (defining the type of the operation) equal to *`Python`* and two
+mandatory attributes *`module`* and *`layer`* in the *`python_param`* dictionary. The *`module`* defines the Python module name
+with the layer implementation, while *`layer`* value is an operation type defined by a user. In order to extract
 attributes for such an operation it is necessary to implement extractor class inherited from the
-`CaffePythonFrontExtractorOp` class instead of `FrontExtractorOp` class used for standard framework layers. The `op`
-class attribute value should be set to the `module + "." + layer` value so the extractor is triggered for this kind of
+*`CaffePythonFrontExtractorOp`* class instead of *`FrontExtractorOp`* class, used for standard framework layers. The *`op`*
+class attribute value should be set to the *`module + "." + layer`* value so the extractor is triggered for this kind of
 operation.
 
-Here is a simplified example of the extractor for the custom operation Proposal from Faster-R-CNN model mentioned above.
-The full code with additional checks is provided in the [https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/tools/mo/openvino/tools/mo/front/caffe/proposal_python_ext.py](https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/tools/mo/openvino/tools/mo/front/caffe/proposal_python_ext.py) file. The sample code uses
-operation `ProposalOp` which corresponds to `Proposal` operation described in the [Available Operations Sets](../../../ops/opset.md)
-document. Refer to the source code below for a detailed explanation of the extractor.
+Below is a simplified example of the extractor for the custom operation Proposal from the mentioned Faster-R-CNN model.
+The full code with additional checks is provided in the [https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/tools/mo/openvino/tools/mo/front/caffe/proposal_python_ext.py](https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/tools/mo/openvino/tools/mo/front/caffe/proposal_python_ext.py) file. 
```

Comment:
```suggestion
The full code with additional checks is provided in the [https://github.com/.../caffe/proposal_python_ext.py](https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/tools/mo/openvino/tools/mo/front/caffe/proposal_python_ext.py) file. 
```

Reply:
```suggestion
The full code with additional checks can be find [here](https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/tools/mo/openvino/tools/mo/front/caffe/proposal_python_ext.py). 
```

Maybe something worth adding to our suggestions file for further discussion, but I don't think that raw links to other files/articles/resources should be present in documentation (full or shortened).

Reply:
I definitely second that idea :)

Reply:
We may discuss it in the spreadsheet, that's a good idea. I don't mind using a short path to a resource, if it is justifiable. I can't imagine using long, full paths, though :) A file name would be fine too, if no additional context is necessary. But I think we should indicate the type of resource the link takes us to. The reader should know what to expect, whether it is an external site, a file, a piece of code, or something.

Reply:
```suggestion
The full code with additional checks can be found [here](https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/tools/mo/openvino/tools/mo/front/caffe/proposal_python_ext.py).
```